src/main/kotlin/net/atos/zac/zoeken/model/FilterParameters.kt

```diff
@@ -4,14 +4,10 @@
  */
 package net.atos.zac.zoeken.model
 
-import jakarta.json.bind.annotation.JsonbProperty
-import nl.lifely.zac.util.AllOpen
 import nl.lifely.zac.util.NoArgConstructor
 
-@AllOpen
 @NoArgConstructor
 data class FilterParameters(
-    @field:JsonbProperty("waarden")
     var values: List<String>,
```

Comment:
can't we convert all properties to `val`s ?

Reply:
Nope.. unfortunately. Or at least, I don't know how. It's because of WildFly's Weld dependency injection in this case I think. And/or JAX-RS. They create beans like this with the default (empty) constructor and then use setters to set the attributes. Very annoying.